frontend/src/queries/examples.ts

```diff
@@ -305,13 +305,16 @@ const HogQLRaw: HogQLQuery = {
           properties.$browser,
           count()
      from events
-    where timestamp > now () - interval 1 day
+    where {filters} -- global date and property filters
       and person.properties.email is not null
  group by event,
           properties.$browser,
           person.properties.email
  order by count() desc
     limit 100`,
+    filters: {
+        dateFrom: '-30d',
```

Comment:
```suggestion
        dateFrom: '-1d',
```

I think we should use `-1d` or `-7d` for the example. This query will get executed quite often when people are just looking around to discover the capabilities of PostHog. Thus it doesn't make sense to have an overly large time window for things people won't look at properly anyway.